lazy_tensor_core/test/test_extract_compiled_graph.py

```diff
@@ -77,6 +78,20 @@ def forward(self, a, b):
         c = a + b
         return a - b, c, a + 1, c
 
+class ModuleWithLTCFallback(nn.Module):
+    """
+    Cover the case that the Fx graph contains LTC fallbacks. In that case,
+    an exception should be thrown.
+    """
+    def __init__(self):
+        super(ModuleWithLTCFallback, self).__init__()
+
+    def forward(self, a):
+        # torch.nonzero cause ltc fallback
+        # TODO: is there a more stable way to force fallback in case LTC implement
```

Comment:
From the code pointer, I see 'force fallback' is controlled by LTC_FORCE_FALLBACK env-variable. I feel it's tricky to update env-variable by the process itself.

Should it looks natural if we maintain a global list of ops (force_forceback_oplist) to force fallback directly? 
- I can add pybind to set that list
- I can keep LTC_FORCE_FALLBACK and if it's set , I'll skip force_forceback_oplist to avoid breaking existing things. 

Does that make sense?

Reply:
yea, i was suggesting to make it a python binding.

Could you try following nick's suggestion here for how to merge a flag into a python binding?
https://fb.quip.com/g6YlAG8nOybT#UNMADApcd7I

i think you could move the 'force fallback' function into either lazy/core/config.cpp or lazy/ts_backend/config.cpp (if its really ts specific) and make it possible to set either ENV or python binding